src/components/AlertsMenu/AlertsMenu.tsx

```diff
@@ -0,0 +1,126 @@
+'use client';
+
+import { Button } from '@nextui-org/button';
+import { Image } from '@nextui-org/image';
+import { Popover, PopoverContent, PopoverTrigger } from '@nextui-org/popover';
+import { useMemo } from 'react';
+
+import { useSidebar } from '../Sidebar/SidebarContext';
+
+export type AlertKey = 'hunger' | 'conflicts1' | 'conflicts2' | 'hazards';
+
+export type AlertTypeKey = 'conflicts';
+
+type Alert = {
+  key: AlertKey;
+  label: string;
+  icon: string;
+};
+
+type AlertType = {
+  key: AlertTypeKey;
+  label: string;
+  icon: string;
+  subalerts: Alert[];
+};
+
+const alertTypes: (Alert | AlertType)[] = [
+  {
+    key: 'hunger',
```

Comment:
keys should generally be Enums

```
export enum AlertTypes { 
HUNGER = 'hunger'
...
]
 ```
 
 
 And use it like this 
 
 `key: AlertType.HUNGER`